build/stage-build-wasm.yml

```diff
@@ -22,7 +22,7 @@
 - task: CopyFiles@2
   displayName: Copy Wasm output
   inputs:      
-    SourceFolder: Uno.Gallery/Uno.Gallery.Wasm/bin/Release/net6.0/dist
```

Comment:
Do we need to change line 14 to 7.0.102?

Reply:
We do :) The build just failed for that. Thanks!